babel-plugin-transform-ember-meta-glob.cjs

```diff
@@ -18,53 +19,65 @@ module.exports = function ({ types: t }) {
         t.stringLiteral(globPath)
       );
     });
-    program.unshiftContainer("body", imports);
+    program.unshiftContainer('body', imports);
 
     const newObj = identifiers.map(({ name, globPath }) => {
       return t.objectProperty(t.stringLiteral(globPath), t.identifier(name));
     });
-    console.log('about to replace')
+    console.log('about to replace');
     path.replaceWith(t.objectExpression(newObj));
   };
   return {
-    name: "ember-meta-glob", // not required
+    name: 'ember-meta-glob', // not required
     visitor: {
       Program(path) {
         // keep a reference to the Program
         program = path;
       },
-      async CallExpression(path) {
+      CallExpression(path, state) {
         const { node } = path;
         // return early if it is not the import we are looking for
         if (
-          node.callee.object?.meta?.name !== "import" ||
-          node.callee.object.property?.name !== "meta" ||
-          node.callee.property?.name !== "glob"
+          node.callee.object?.meta?.name !== 'import' ||
+          node.callee.object.property?.name !== 'meta' ||
+          node.callee.property?.name !== 'glob'
         ) {
           return;
         }
 
         // get the files from the file system
         const pathName = node.arguments[0].value;
-        let foundFiles = await glob(pathName, { ignore: "node_modules/**" });
+        let foundFiles = globSync(pathName, {
+          ignore: 'node_modules/**',
+          cwd: dirname(
+            [
+              ...state.cwd.split('/'),
+              'app',
+              ...state.filename
+                .slice(state.cwd.length + 1)
+                .split('/')
+                .slice(1),
+            ].join('/')
```

Comment:
this whole thing is just because it looks like ember-cli-babel is actually reporting the wrong file to us 😭 lots of hoops to jump through just to get a real path